lib/tsconfig/TsConfigGrammar.c

```diff
@@ -71,6 +71,7 @@
 
 # include "TsConfigParseEvents.h"
 # include "tscore/ink_defs.h"
+# include "TsConfigGrammar.h"
 
 // Types we need for the lexer.
```

Comment:
Isn't this a generated file?  As I recall there are odd reasons it needs to be checked in.

Reply:
this one isnt generated afaik, however TsConfigGrammer.hpp is generated

Either way this fix didnt work and just made all the builds fail in the same way the fedora one does. This was just a quick check from discussions with @jrushford since this is the easiest way for us to try a freebsd build

Reply:
This is generated from "TsConfigGrammar.y". See `BUILD_TSCONFIG_GRAMMAR` from "configure.ac".

How does this fail in FreeBSD?

Reply:
`In file included from TsBuilder.cc:27:
./TsConfigGrammar.hpp:21:16: error: ISO C++ forbids forward references to 'enum' types
  typedef enum yytokentype yytoken_kind_t;
               ^
1 error generated.
gmake[3]: *** [Makefile:826: TsBuilder.lo] Error 1`